src/App.ts

```diff
@@ -21,7 +20,7 @@ export class App {
   }
 
   setupCommandParser(){
```

Comment:
nit: setUpCommandParser(), same for below functions

Reply:
also, why is the command parser initialized in App's ctor? Doesn't the parser depend on what prefix the user includes in the message?

Reply:
Parser is now static class.